src/server/config/schema.js

```diff
@@ -161,6 +161,13 @@ module.exports = () => Joi.object({
   status: Joi.object({
     allowAnonymous: Joi.boolean().default(false)
   }).default(),
+  map: Joi.object({
+    manifestServiceUrl: Joi.when('$dev', {
```

Comment:
this is a new internal (undocumented) setting. serves both the coordinate map and the region map.